### PR TITLE
March newsletter

### PR DESCRIPTION
**Pull request by davidfischer**

### CODE REVIEW

File: content/posts/newsletter-march-2024.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+* We released a feature to target campaigns by the day of the week.
+  This was a minor feature but a couple advertisers wanted to run only weekend campaigns.
+  We've discussed it in the past, but our network's traffic on weekends
+  is less than half a normal weekday.
+* We've made a lot of progress toward some big changes in our
+  [machine learning based contextual targeting]({filename}../posts/2022-content-based-targeting.md).
+  These changes will let us target a lot of niches that were previously too hard or too small to target.
+  We are still aiming for a public demo this month!
+* We are experimenting with some self-serve features to let advertisers
+  who know exactly what they want to start a campaign without doing a call with a co-found^D^D^D sales rep.
```
Comment (ericholscher) [inline content/posts/newsletter-march-2024.md:27]: Not sure our audience is gonna get the `^D` reference. Is there markdown for strikethrough? 

Also seems like this is a small nod to hiding Ras, but might be worth a line item?

**Review by ericholscher — APPROVED** [review]
Looks solid 👍

File: content/posts/newsletter-march-2024.md (hunk 1 above)
Comment (ericholscher) [reply content/posts/newsletter-march-2024.md:27]:
```suggestion
  who know exactly what they want to start a campaign without doing a call with a ~~co-founder~~ sales rep.
```

Comment (davidfischer) [reply content/posts/newsletter-march-2024.md]: I ended up using the `<s>` tag and I added a brief blurb for Raz.

**Pull request merged**